gitpod-io/website
# Use F1 for VS Code Browser command palette

## Description

![image](https://github.com/gitpod-io/website/assets/8781107/8dd335f1-6dab-40b0-9e05-dac6309d88a0)

Ctrl+Shift+P on Firefox opens private window.

## Related Issue(s)



Fixes #

## How to test

1. Open VS Code Browser
2. Press <kbd>Ctrl+Shift+P</kbd>. see new windows pops up, close it
3. Press <kbd>F1</kbd>, see command palette

## Release QA Checklist ✔️



<details>
  <summary>QA Checklist </summary>

## [Priority: High] Functional Testing

  

## [Priority: High] Content Testing


## [Priority: Medium] Performance Testing


## [Priority: Medium] Accessibility Testing


By including these key items in your website release QA checklist, you can ensure that your website is fully functional, user-friendly, and meets your business requirements.

## Security Testing

> **Note**:💡 If we are changing something related to cookies or data storing or analytics side.


  </details>


<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3723"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
